namespaces/cloud-platform-live-0.k8s.integration.dsd.io/nickp-test-dev/resources/ecr.tf

```diff
@@ -9,7 +9,7 @@ provider "aws" {
 module "ecr-repo" {
   source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.1"
   repo_name = "nickp-test-dev"
-  team_name = "tactical-products-np"
+  team_name = "tactical-products"
```

Comment:
This change will delete the previous repository `.../tactical-products-np/nickp-test-dev`, and create the new one.

Reply:
That's no problem as it's just a test repo.  Thanks for the warning though :-)